molgenis-data-security/src/main/java/org/molgenis/data/security/meta/EntityTypeRepositorySecurityDecorator.java

```diff
@@ -330,6 +333,7 @@ private void deleteEntityPermissions(String entityTypeId)
 	public void add(EntityType entity)
 	{
 		validateAddAllowed(entity);
+		createAcl(entity);
```

Comment:
Tricky! `validateAddAllowed` currently checks if you have the WRITE_META permission on the entity. Acl gets created after the check.